src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -183,6 +196,25 @@ export default {
             }
             return false
         },
+        tooltipContent() {
+            return this.$t('loading_external_layer')
+        },
+    },
+    watch: {
+        currentLang() {
+            this.tippyInstance.setContent(this.tooltipContent)
+        },
+    },
+    mounted() {
+        this.tippyInstances = tippy('#loadingButton', {
+            content: this.tooltipContent,
+            arrow: true,
+            placement: 'top',
+            touch: false,
+        })
+    },
+    beforeUnmount() {
+        this.tippyInstances?.forEach((tooltip) => tooltip.destroy())
```

Comment:
The issue with the `ref` approach is that we have this warning at first application load 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/cfd81dc8-9cb3-468a-8dd9-b9e16e68d5ab)

Reply:
Also using `ref` doesn't necessary return one instance because it is attached to a list of elements, to use a `ref` as for the third party disclaimer I would have to put the load button in a separate vue component.

Reply:
changed to the class instead of id